src/graphs.py

```diff
@@ -1,23 +1,12 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
-import gi
 import os
-import re
```

Comment:
Quite a large amount of differences in this file, but that seems to be a consequence from moving away many functions to their own new files. From what I've looked at it seems good.